packages/language-server/src/prisma-fmt/textDocumentCompletion.ts

```diff
@@ -1,11 +1,16 @@
-import prismaFmt from '@prisma/prisma-fmt-wasm'
+import { prismaFmt } from '../wasm'
 import { CompletionParams, CompletionList } from 'vscode-languageserver'
+import { handleWasmError } from './util'
 
 /* eslint-disable @typescript-eslint/no-explicit-any,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-return */
 
 // This can't panic / throw exceptions. Any panic here should be considered a
 // bug to be fixed. prisma-fmt will return an empty CompletionList on error.
-export default function textDocumentCompletion(schema: string, params: CompletionParams): CompletionList {
+export default function textDocumentCompletion(
+  schema: string,
+  params: CompletionParams,
+  onError?: (errorMessage: string) => void,
```

Comment:
`onError` -> `showErrorToast`

Reply:
My thought here for not renaming this to `showErrorToast` was that it then felt like an implementation detail. So, looking at this section in `MessageHandler.ts`, the way it reads to me is, `textDocumentCompletion` can do something `onError` and we tell it what specifically, so in this case, `onError` -> `showErrorToast` but that could be something else, `onError` -> `otherThing`

```ts
const completionList = textDocumentCompletion(text, params, (errorMessage: string) => {
    if (showErrorToast) {
      showErrorToast(errorMessage)
    }
  })
  ```

Reply:
Yes it totally does - and is what I realized at https://github.com/prisma/language-tools/pull/1396#discussion_r1142036300.

So for me the function parameter should almost always be called `onError` as that takes _any_ function that does something with an error string, and the function you then define and use when calling this function can be called the way